patches/chrome-browser-ui-views-update_recommended_message_box.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/update_recommended_message_box.h b/chrome/browser/ui/views/update_recommended_message_box.h
+index 7c112c68763fd09b7ed563d2eda881112bb22ef5..4dbef625579bf0919b033181f3556d8469b33342 100644
+--- a/chrome/browser/ui/views/update_recommended_message_box.h
++++ b/chrome/browser/ui/views/update_recommended_message_box.h
+@@ -20,6 +20,8 @@ class UpdateRecommendedMessageBox : public views::DialogDelegate {
+   static void Show(gfx::NativeWindow parent_window);
+ 
+  private:
++  friend class UpdateRecommendedMessageBoxMac;
```

Comment:
which private member do we need to access?

Reply:
Its ctor/dtor private. So, it was needed to subclass.

Reply:
we don't need forward declaration?

Reply:
Yep, build was fine on windows/mac.
https://stackoverflow.com/questions/4492062/why-does-a-c-friend-class-need-a-forward-declaration-only-in-other-namespaces